assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -538,6 +538,9 @@ che.infra.openshift.trusted_ca.dest_configmap_labels=config.openshift.io/inject-
 # Additional labels to add into every Route created by Che server
 # to allow clear identification.
 che.infra.openshift.route.labels=NULL
+# The hostname that should be used as a suffix for the workspace routes.
+# For example host=open.che.org then the route will look like routed3qrtkptr0.open.che.org
```

Comment:
host of the Route has some length limitations. It would be useful to write here some warning, what is maximum length of the domain suffix.